src/action/mod.rs

```diff
@@ -272,6 +272,8 @@ pub enum ActionError {
     PathGroupMismatch(std::path::PathBuf, u32, u32),
     #[error("`{0}` exists with a different mode ({1:o}) than planned ({2:o}), consider updating it with `chmod {2:o} {0}`")]
```

Comment:
We should also improve this error message to suggest a valid command for the user... That may just be going and setting the modes to be `0o0644` instead of `0o100644` where we do pass it.